jena-arq/src/main/java/org/apache/jena/riot/system/RiotLib.java

```diff
@@ -73,12 +78,37 @@ public static Node createIRIorBNode(String iri)
         return NodeFactory.createURI(iri) ;
     }
 
-    /** Test whether  */
-    public static boolean isBNodeIRI(String iri)
-    {
+    /** Test whether a IRI is a ARQ-encoded blank node. */
+    public static boolean isBNodeIRI(String iri) {
         return skolomizedBNodes && iri.startsWith(bNodeLabelStart) ;
     }
     
+    // These two must be in-step.
+    /** Function applied to undefined prefixes to convert to a URI string */  
+    public static final Function<String,String> fixupPrefixes      = (x) -> "::"+x ;
```

Comment:
I'm probably missing something here, but it seems like this is only ever used unwrapped, via `fixupPrefixes.apply`, and that's down below in this class (where it is used instead of using `fixupPrefixIRI(String)`, but I'm not sure why). Maybe it would be clearer to have an ordinary method and in future, if a function is needed, to use a method reference?